src/hooks/useMarketsData.ts

```diff
@@ -229,6 +234,7 @@ export const useMarketsData = ({
               tickSizeDecimals,
               trades24H: 0,
               volume24H: 0,
+              isFavorite: favoritedMarkets.includes(`${id}-USD`),
```

Comment:
rip I wish we had a helper somewhere like `getMarketIdFromAssetName`?

Reply:
yeah this does seem like a recipe for a very subtle bug in the future. Can we just do the 100% correct thing and look it up in our market map?

Reply:
@tyleroooo the MarketMap only provides assets. It doesn't have any notion of markets so we do have to append the quote asset (USDC). I think what we can do is pull the `${id}-USD` outside of the safe-assign so we can use in `id` as well.

Reply:
oooh yeah I can see id getting set above, yeah this is fine